bitsofgood/pencils-for-success
# feat(ui): add admin information modal

### Description

- Add Credentials information modal using react context providers and controllers.
- Added username and password validation
- Add submit confirmation message
- Added view and edit credentials feature to both admin and chapter navbars

**Related Issues/PRs:** 
- Closes #132 
- Closes #142 

### Test Plan
- Login as admin
- Click on "View Admin Info" in top navbar
- Click on "Edit Admin"
- Test Username Validation (Alphanumeric, Greater than 6 characters, less than 31 characters)
- Test Password Validation (Greater than 8 characters, less than 129 characters)
- Click save and confirm the confirmation modal is visible
- Repeat process for chapters
